apps/klasses/views.py

```diff
@@ -48,6 +49,12 @@ class OverView(LoginRequiredMixin, DetailView):
 
     def get_context_data(self, **kwargs):
         context = super(OverView, self).get_context_data(**kwargs)
+        try:
+            klass = kwargs.get('object')
+        except ObjectDoesNotExist:
+            raise Http404
+
+        context['completely_graded'] = klass.homeworks_completely_graded()
```

Comment:
`WizardMixin` is where we set all of these context variables for the `steps`

Reply:
Great call. Thank you!